### ISSUE TITLE
Non latin characters break web LCD

### ISSUE THREAD

**Issue by dodther** [vitotai/BrewManiacEsp8266#59]
![2019-12-09_12-56-15](https://user-images.githubusercontent.com/15664548/70403698-5954f780-1a83-11ea-99cf-e465345aadfe.png)
![2019-12-09_12-48-49](https://user-images.githubusercontent.com/15664548/70403712-62de5f80-1a83-11ea-9e32-4e6c18209cd5.png)

**Comment by vitotai** [comment]
Obviously, that character is not in the FONT library of Javascript. Does your LCD show the character?

**Comment by dodther** [comment]
My LCD show all characters.

**Comment by vitotai** [comment]
[bm.english.htm.gz](https://github.com/vitotai/BrewManiacEsp8266/files/3945410/bm.english.htm.gz)

If you are using 0.4.8 HTML/Javascript, you might try this file by uploading as "bm.htm.gz".

**Comment by dodther** [comment]
File upload didn’t help.

**Comment by vitotai** [comment]
Did you force reloading the page? Although the file I provided doesn't contains cyrillic font but European font, you should see something else.
I don't have the font data of cyrillic right now. A way to get font data must be developed unless the font library is provided.

**Comment by dodther** [comment]
I reloading the page. Can I help with fonts?

**Comment by vitotai** [comment]
[bm.htm.gz](https://github.com/vitotai/BrewManiacEsp8266/files/3960717/bm.htm.gz)

Sorry, I think I gave a wrong file. Try this one with cyrillic fonts.
